scripts/us_cdc/environmental_health_toxicology/schema.mcf

```diff
@@ -0,0 +1,93 @@
+Node: dcid: MeanConcentrationAirPollutantPM2.5StandardError
+name: "MeanConcentrationAirPollutantPM2.5StandardError"
+description: "Standard error for mean PM2.5 concentration in air."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Atmosphere
+measuredProperty: dcs:concentration
+pollutant: dcs:PM2.5
+statType: dcs:stdError
+
+Node: dcid: MeanConcentrationAirPollutantOzoneStandardError
+name: "MeanConcentrationAirPollutantOzoneStandardError"
+description: "Standard error for mean Ozone concentration in air."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Atmosphere
+measuredProperty: dcs:concentration
+pollutant: dcs:Ozone
+statType: dcs:stdError
+
+Node: dcid:MedianConcentrationAirPollutantOzone
+name: "MedianConcentrationAirPollutantOzone"
+description: "Median ozone concentration in air."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Atmosphere
+measuredProperty: dcs:concentration
+statType: dcs:medianValue
+pollutant: dcs:Ozone
+
+Node: dcid:MedianConcentrationAirPollutantPM2.5
+name: "MedianConcentrationAirPollutantPM2.5"
+description: "Median PM2.5 concentration in air."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Atmosphere
+measuredProperty: dcs:concentration
+statType: dcs:medianValue
+pollutant: dcs:PM2.5
+
+Node: dcid:PopulationWeightedConcentrationAirPollutantOzone
+name: "PopulationWeightedConcentrationAirPollutantOzone"
+description: "Population-weighted ozone concentration in air."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Atmosphere
+measuredProperty: dcs:concentration
+statType: dcs:measuredValue
+pollutant: dcs:Ozone
+measurementQualifier: dcs:CDCPopulationWeighted
+
+Node: dcid:PopulationWeightedConcentrationAirPollutantPM2.5
+name: "PopulationWeightedConcentrationAirPollutantPM2.5"
+description: "Population-weighted ozone concentration in air."
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Atmosphere
+measuredProperty: dcs:concentration
+statType: dcs:measuredValue
+pollutant: dcs:PM2.5
+measurementQualifier: dcs:CDCPopulationWeighted
+
+Node: dcid:CDCPopulationWeighted
+name: "CDCPopulationWeighted"
+description: "Measurement is calculated for each county based on census tract estimates and 2010 U.S. Census census tract-level population data."
+typeOf: dcs:MeasurementQualifierEnum
+
+Node: dcid:precipitation
+name: "precipitation"
+typeOf: schema:Property
+domainIncludes: dcs:Atmosphere
+rangeIncludes: Float
+
+Node: dcid:PrecipitationIndexEnum
+name: "PrecipitationIndexEnum"
+subClassOf: schema:Enumeration
+typeOf: schema:Class
+description: "An enumeration of different precipitation indices."
+
+Node: dcid:PalmerDroughtSeverityIndex
+name: "PalmerDroughtSeverityIndex"
+domainIncludes: dcs:Atmosphere
+typeOf: dcs:PrecipitationIndexEnum
+rangeIncludes: schema:Float
```

Comment:
rangeIncludes and domainIncludes are specified for Property instances. This being an instance of PrecipitationIndexEnum, they aren't required.

So, what is the property that PrecipitationIndexEnum values get assigned to? (i.e., the property whose rangeIncludes is PrecipitationIndexEnum)